routes/nodeMailer.js

```diff
@@ -0,0 +1,29 @@
+const express = require('express');
+const transporter = require('../transporter');
+// const verifyToken = require('../services/authService');
```

Comment:
Instead of this comment, can you make a `TODO` note to add `verifyToken` to the route once auth is implemented? It can be something like `// TODO: add verifyToken`